.readme-partials.yml

```diff
@@ -1,4 +1,4 @@
-title: |-
```

Comment:
Just slightly confused, does removing the title here add it somewhere else?

Reply:
I believe the previous version overwrites title with an empty title. Removing `title` actually adds the title in. But really I'm just following the other repos' partial files where the title displays correctly.

Reply:
Sounds good! Thank you :)